server/src/database/database.module.ts

```diff
@@ -51,10 +54,16 @@ const appOrmConfig: PostgresConnectionOptions = {
 @Module({
   imports: [
     TypeOrmModule.forRoot(appOrmConfig),
-    TypeOrmModule.forFeature([Application, FormMetaData]),
-    UserModule,
+    TypeOrmModule.forFeature([Application, FormMetaData, Comment, User]),
+  ],
+  providers: [
+    Logger,
+    SyncChefsDataService,
+    ApplicationService,
+    FormMetaDataService,
+    CommentService,
+    UserService,
```

Comment:
Any reason to add this here?

Reply:
Blows up without it :(
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/102998728/205686656-a27fc9f2-c7cf-4d09-9c02-7a9ef944ab68.png">